.circleci/config.yml

```diff
@@ -1107,6 +1116,8 @@ workflows:
             branches:
               only:
                 - master
+      - changelog-check:
+          context: team-consul
```

Comment:
I guess this means that the check will only work for internal contributions, is that right?

I think it is equally important to have this check run on PRs from external contributors so that we don't have to ask them every time to add a changelog entry.

We should be able to accomplish that by using github actions to do this work instead of circleCI. If we use `pull_request_target` (https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target), which will run using the code from the master branch.

This should allow us to run the check for external contributions as well.